applications/nrf_desktop/README.rst

```diff
@@ -88,6 +88,115 @@ Testing
+* desktop mice - press the scroll (the scroll has to be pressed before the mouse is powered up).
+* keyboard - Page Down key with Fn modifier key.
+* dongle - the "SW1" button.
+
+Gaming mouse also has dedicated hardware switch located at the bottom.
+The switch is used to select between:
+
+* dongle peer - top position.
+* other BLE peers - middle position.
+* mouse turned off - bottom position.
+
+The following peer control operations are available:
+
+* **peer selection**
+
+  - Peripheral supporting multiple peers (the gaming mouse, the keyboard)
+
+    Short click to initialize peer selection.
+    During the peer selection short click selects the next peer and double click confirms the selection.
+    The peer is changed after the confirmation.
+
+    For the gaming mouse the dedicated hardware switch is used to select the dongle peer.
+    When the dongle peer is selected, the peer control is disabled until the switch will be set to another position.
+
+* **bond erase**
+
+  Long click to initialize peer erase, then double click to confirm the operation.
+
+  - Central (the dongle)
+
+    After the confirmation all the Bluetooth bonds are removed.
+
+  - Peripheral with dedicated key used only for the peer control (the gaming mouse, the keyboard)
+
+    After the confirmation Bluetooth advertising using new local identity is started.
+    When a new Bluetooth Central successfully connects and establishes security, the old bond is removed and the new bond is used instead of it.
+    If a new peer does not connect in predefined amount of time, the advertising ends and the application switches back to the old peer.
+
+  - Peripheral with no dedicated key for the peer control (the desktop mice)
+
+    The desktop mice do not have dedicated key used for peer control.
+    Long press of the scroll must be started before the mouse is turned on.
+    The confirmation is not needed.
+
+* **start scan**
+
+  - Central (the dongle)
+
+    Short click to instantly start scanning for both bonded and not bonded Bluetooth Peripherals.
+    The scan will be interrupted if another dongle connected device is in use.
+
+* **cancel operation**
+
+  - all the nRF52 Desktop devices
+
+    Press the button for time longer than short click, but shorter than long click to cancel ongoing peer operation.
+
+During the Bluetooth peer operations visual feedback is provided using LED (if the device has LEDs).
+Each of the states is represented by separate LED color and effect.
+
+Power Management
+================
+
+Reducing power consumption is important for every battery powered device.
+The nRF52 Desktop Peripherals are suspended (if BLE connected) or powered off (if not connected via BLE) when they are not in use for predefined amount of time.
+In that case most of the functionalities are disabled - for example LEDs are turned off and advertising is stopped.
+Moving mouse or pressing any button will wake up the device and turn on the disabled functionalities.
```

Comment:
In suspended state the device will maintain the active connection while in powered off state CPU will be switched to OFF mode.

When mouse is powered from the USB power down timeout is disabled.

Reply:
done